internal/cloud/cloudplan/saved_plan.go

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) HashiCorp, Inc.
 // SPDX-License-Identifier: MPL-2.0
-
 package cloudplan
 
 import (
 	"encoding/json"
+	"errors"
+	"io"
 	"os"
+	"strings"
 )
 
+var ErrInvalidRemotePlanFormat = errors.New("invalid remote plan format, must be 1")
+var ErrInvalidRunID = errors.New("invalid run ID")
```

Comment:
One potential option here is re-use: [tfe.ErrInvalidRunID](https://github.com/hashicorp/go-tfe/blob/11175f4aeee6f6d1ee8e3170f98bde61cc0e4b23/errors.go#L103)